### PR TITLE
Automation poc

### PR DESCRIPTION
**Pull request by charles-testco**
### Summary:
This PR implements two automation frameworks: Playwright and Cypress. 
Playwright was already introduced but failing and not being. We have fixed and extended the tests.
Cypress is new to the project.
A README-E2E.md has been created to explain how to run the tests

### Checklist:
- [x] I merged latest `<base branch>`
- [x] I manually verified the change
- [ ] I added labels to my PR
- [ ] I tested in multiple browsers
- [ ] I added relevant unit tests
- [ ] I have notified others of changes they need to make locally (migrations, jobs, package updates, etc)

### CODE REVIEW

File: src/frontend/cypress/support/e2e.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
+/// <reference types="cypress" />
```
Comment (mmmmmmaya) [inline src/frontend/cypress/support/e2e.ts]: what's happening in this file?

File: src/frontend/cypress/utility/auth.ts [hunk 2]
```diff
@@ -0,0 +1,25 @@
+/// <reference types="cypress" />
```
Comment (mmmmmmaya) [inline src/frontend/cypress/utility/auth.ts]: this looks very similar to the command added to cypress in support/commands. What's the difference/why is there dupilication?

File: src/frontend/playwright/setup/staging.config.ts [hunk 3]
```diff
@@ -0,0 +1,32 @@
+import { PlaywrightTestConfig } from "@playwright/test";
+import { devices } from '@playwright/test';
+import dotenv from 'dotenv';
+
+dotenv.config();
+
+const config: PlaywrightTestConfig = {
+  testDir: '../tests',
+  timeout: 30000,
```
Comment (mmmmmmaya) [inline src/frontend/playwright/setup/staging.config.ts:9]: why does playwright have a shorter timeout than cypress?

File: src/frontend/playwright/storage/state.json [hunk 4]
```diff
@@ -0,0 +1,69 @@
+{
```
Comment (mmmmmmaya) [inline src/frontend/playwright/storage/state.json]: what do these cookies do?

File: src/frontend/package.json [hunk 5]
```diff
@@ -89,7 +96,15 @@
...
     "babel-plugin-transform-imports": "^2.0.0",
+<<<<<<< HEAD
+    "concurrently": "^6.0.0",
+    "cypress": "^10.3.0",
+    "cypress-localstorage-commands": "^1.6.1",
+    "cypress-wait-until": "^1.7.2",
+    "cypress-file-upload": "^5.0.8",
+=======
     "concurrently": "^6.5.1",
+>>>>>>> trunk
```
Comment (phoenixAja) [inline src/frontend/package.json]: some merging relics left behind

File: src/frontend/playwright/tests/home.spec.ts [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+    //"Privacy":  "/privacy",
+    //"Terms":  "/terms",
+    //"Contact Us":  "mailto:hello@czgenepi.org"
+}
+
+test.describe("Home page tests", () => {
+  test('Should verify home page', async ({ page }) => {
+    await page.goto('https://staging.czgenepi.org');
+    await expect(page.locator(getByTestID("navbar-landing"))).not.toBeEmpty();
+    await expect(page.locator(getByTestID("navbar-sign-in-link"))).toBeVisible();
```
Comment (ehoops-cz) [inline src/frontend/playwright/tests/home.spec.ts:17]: What is the difference between `.not.toBeEmpty()` and `.toBeVisible`?

Comment (mmmmmmaya) [reply src/frontend/playwright/tests/home.spec.ts:17]: I think `toBeEmpty` implies that it doesn't have children elements or text, whereas `toBeVisible` implies you can see it on the page.

**Comment by mmmmmmaya** [comment]
is this PR still needed? if not let's close it

**Comment by charles-testco** [comment]
This PR has been superceded

**Pull request closed without merging**